src/components/modal/ModalBackdrop.tsx

```diff
@@ -39,4 +57,12 @@ export class ModalBackdrop extends React.Component<IModalBackdropProps, {}> {
             </div>
         );
     }
+
+    private onKeyDown = (e: KeyboardEvent) => {
+        if (this.canClose && e.code === 'Escape') {

```

Comment:
should be `e.keyCode === keyCode. escape`

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code#Browser_compatibility

Reply:
I will use key instead (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key) since keyCode is deprecated and cannot even be used in the tests.